gradle/formatting.gradle

```diff
@@ -10,6 +10,13 @@ allprojects {
             eclipse().configFile rootProject.file('formatter/formatterConfig.xml')
             trimTrailingWhitespace()
 
+            custom 'Refuse wildcard imports', {
+                // Wildcard imports can't be resolved; fail the build
+                if (it =~ /\s+import .*\*;/) {
```

Comment:
```suggestion
                if (it =~ /\s*import.*\*;/) {
```

Reply:
Imports nearly always start after a new line and the `\s+` was in preference to `[\n|\r|\n\r]\s*`.  There should always be a `package` declaration prior to the imports, although I'm not sure we have a style check to enforce declaring a package.

I think `import*\*;` would behave identically to `\s*import.*\*;` and don't think that's what we want.